src/main/java/core/basesyntax/services/operations/Return.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.services.operations;
+
+import java.util.Map;
+
+public class Return implements Operable {
+    private final Supply supply;
+
+    public Return(Supply supply) {
+        this.supply = supply;
+    }
+
+    @Override
+    public boolean updateStorage(Map<String, Map<String, Integer>> store, String[] data) {
+
+        return supply.updateStorage(store, data);
+    }
```

Comment:
this class is redundant, cause it doesn't contain any kind of logic, just remove it and initialize your map with key - "r", valuse "SupplyOperation"